rails_application/test/customers/customer_promoted_to_vip_test.rb

```diff
@@ -0,0 +1,39 @@
+require "test_helper"
+
+module Customers
+  class CustomerPromotedTovipTest < InMemoryTestCase
+    cover "Customers"
+
+    def setup
+      super
+      Customer.destroy_all
+    end
+
+    def test_promote_customer_to_vip
+      event_store = Rails.configuration.event_store
+
+      customer_id = SecureRandom.uuid
+      run_command(
+          Crm::RegisterCustomer.new(
+            customer_id: customer_id,
+            name: "Joe Fake"
+          )
+      )
+
+      event_store.publish(
```

Comment:
I left both publish and run command to ask this question there:

In those unit tests, we are teting if our event published will cause customer to have changed Vip flag. 
We don't test running command, because publishing event by command is tested on CRM side and now we just see if proper event is causing proper outcome yea ?